api/services/connect/workflows/paths/index.yaml

```
@@ -14,3 +14,33 @@ get:
   tags:
   - Connect Workflows
 parameters: []
+post:
+  operationId: post_workflow_list
+  parameters:
+  - $ref: "../parameters/workspace_id.yaml"
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: "../../schemas/WorkflowInput.yaml"
+    required: true
+  responses:
+    '200':
+      content:
+        application/json:
+          schema:
+            $ref: "../../schemas/WorkflowModel.yaml"
+      description: Success
+    '201':
+      content: {}
+      description: Workflow created successfully
+    '400':
+      content: {}
+      description: Request validation error
+    '409':
+      content: {}
+      description: Duplicate workflow
+  summary: Create a new workflow by creating a new SSM automation document in AWS
```

No need to mention the details about SSM document in the public API doc.
```suggestion
  summary: Create a new workflow in the selected workspace.
```